src/components/SharedComponents/PhotoCount.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import { View } from "components/styledComponents";
+import * as React from "react";
+import Svg, { Path, Text } from "react-native-svg";
+
+type Props = {
+  count: number,
+  size?: number,
+  shadow: boolean,
+};
+
+const PhotoCount = ( { count, size, shadow }: Props ): React.Node => {
+  if ( count <= 0 || count > 99 ) {
+    return null;
+  }
+
+  return (
+    <View
+      className={`drop-shadow-md w-[${String( size )}px] h-[${String( size )}px] ${String(
```

Comment:
We are using https://www.npmjs.com/package/classnames for more optional combinations of classNames. So, e.g. you can refactor to add `shadow && "shadow"` as one argument to the classNames fct.

Reply:
Also, shadow should have 0.25 instead of 0.1 for alpha channel

Reply:
I think that [NativeWind](https://www.nativewind.dev/)  dont support all drop shadow as in CSS
I used for the shadow react native styles

Reply:
Okay, thanks for checking. We will have to change back for everything else here though, see new inline comment.